src/plugins/discover/public/application/components/default_discover_table/default_discover_table.tsx

```diff
@@ -247,7 +247,8 @@ const DefaultDiscoverTableUI = ({
               (row: OpenSearchSearchHit, index: number) => {
                 return (
                   <TableRow
-                    key={row._id}
+                    // TODO: use the id from the row._id when all languages support retreving it
+                    key={index}
```

Comment:
how about `key={row._id || index}`

Reply:
Using existing row._id could improve efficiency of re-creating TableRow component when re-rendering. It's totally fine we will fix it soon, so every row has a unique id.